spec/models/book_csv_importer_spec.rb

```diff
@@ -0,0 +1,117 @@
+require "rails_helper"
+
+describe BookCsvImporter do
+  before do
+    Book.delete_all
+    Genre.delete_all
+    Author.delete_all
+  end
```

Comment:
This caught us unawares. We both totally forgot this was necessary.

That communicates to me that we should make this more of an automatic process.

We could use the database cleaner gem: https://github.com/DatabaseCleaner/database_cleaner

...But I think it might be useful to write your own rudimentary implementation of that first.

Let's talk about that today.